docs/website/assets/sass/docs.sass

```diff
@@ -15,7 +15,6 @@ $panel-header-logo-width: 85%
   +row
   position: relative
   overflow: hidden
-  height: 100vh
```

Comment:
I don't think this is needed anyway... but now we're displaying something outside the dashboard we can't use this without getting an extra scrollbar in some browsers.